<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Перестройка</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1300</width>
      <height>640</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="menuWindow">
     <widget class="QPushButton" name="shutdownWindow">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>380</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectGameButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>390</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Подключиться</string>
      </property>
     </widget>
     <widget class="QPushButton" name="runGameButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>300</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QLabel" name="background1">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>-100</y>
        <width>620</width>
        <height>630</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="regulationsButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>190</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Правила</string>
      </property>
     </widget>
     <widget class="QLabel" name="mainIMG">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>500</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>background1</zorder>
     <zorder>shutdownWindow</zorder>
     <zorder>connectGameButton</zorder>
     <zorder>runGameButton</zorder>
     <zorder>regulationsButton</zorder>
     <zorder>mainIMG</zorder>
    </widget>
    <widget class="QWidget" name="runGameWindov">
     <widget class="QPushButton" name="backMenuWindow">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startGameButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>500</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Начать Перестройку</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nicknameEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>500</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="nicknameEditButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>60</y>
        <width>200</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сменить имя</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="listPlayers">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>290</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="background2">
      <property name="geometry">
       <rect>
        <x>-100</x>
        <y>-80</y>
        <width>700</width>
        <height>600</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="listCards">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>290</width>
        <height>150</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Состав карт&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#00ff7f;&quot;&gt;1. Предприниматель&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;2. Киллер&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ffff00;&quot;&gt;3. Телеведущий&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;4. Бюрократ&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;5. Диссидент&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>background2</zorder>
     <zorder>backMenuWindow</zorder>
     <zorder>startGameButton</zorder>
     <zorder>nicknameEdit</zorder>
     <zorder>nicknameEditButton</zorder>
     <zorder>listPlayers</zorder>
     <zorder>listCards</zorder>
    </widget>
    <widget class="QWidget" name="gameWindow">
     <widget class="QPushButton" name="playerCard1BTN">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>460</y>
        <width>101</width>
        <height>145</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="playerCard2BTN">
      <property name="geometry">
       <rect>
        <x>354</x>
        <y>460</y>
        <width>101</width>
        <height>145</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="oponent1Card1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>-40</y>
        <width>110</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="oponent1Card2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>-40</y>
        <width>110</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="oponent2Card2">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>-40</y>
        <width>110</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="oponent2Card1">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>-40</y>
        <width>110</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="iconCoins">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>580</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;iconCoins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="countCoins">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>580</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;29&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="namePlayer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Player1234&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusPlayer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>250</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;STATUS&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;STATUS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameOponent1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Player2341&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="countCoinsOponent1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;29&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="iconCoinsOponent1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;iconCoins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusOponent1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>190</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;STATUS&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;STATUS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusOponent2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>30</y>
        <width>190</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;STATUS&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;STATUS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameOponent2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>0</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Player3412&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="countCoinsOponent2">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;29&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="iconCoinsOponent2">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;iconCoins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="countCoinsOponent3">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;29&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="iconCoinsOponent3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;iconCoins&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameOponent3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>0</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Player4123&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="oponent3Card1">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>-40</y>
        <width>110</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="oponent3Card2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>-40</y>
        <width>110</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusOponent3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>190</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;STATUS&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;STATUS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerCard2">
      <property name="geometry">
       <rect>
        <x>357</x>
        <y>463</y>
        <width>110</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerCard1">
      <property name="geometry">
       <rect>
        <x>253</x>
        <y>463</y>
        <width>110</width>
        <height>140</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CARD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="card5BTN">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>460</y>
        <width>300</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Диссидент</string>
      </property>
     </widget>
     <widget class="QPushButton" name="card1BTN">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>510</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Предприниматель</string>
      </property>
     </widget>
     <widget class="QPushButton" name="card3BTN">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>560</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Телеведущий</string>
      </property>
     </widget>
     <widget class="QPushButton" name="card2BTN">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>510</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Киллер</string>
      </property>
     </widget>
     <widget class="QPushButton" name="card4BTN">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>560</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Бюрократ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="restructuringBTN">
      <property name="geometry">
       <rect>
        <x>846</x>
        <y>530</y>
        <width>194</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Перестройка</string>
      </property>
     </widget>
     <widget class="QPushButton" name="takeOneMoneyBTN">
      <property name="geometry">
       <rect>
        <x>846</x>
        <y>460</y>
        <width>194</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Взять 1 монету</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkBTN">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>460</y>
        <width>240</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ПРОВЕРИТЬ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="notCheckBTN">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>530</y>
        <width>240</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Не проверять</string>
      </property>
     </widget>
     <zorder>oponent1Card1</zorder>
     <zorder>oponent1Card2</zorder>
     <zorder>oponent2Card2</zorder>
     <zorder>oponent2Card1</zorder>
     <zorder>iconCoins</zorder>
     <zorder>countCoins</zorder>
     <zorder>namePlayer</zorder>
     <zorder>statusPlayer</zorder>
     <zorder>nameOponent1</zorder>
     <zorder>countCoinsOponent1</zorder>
     <zorder>iconCoinsOponent1</zorder>
     <zorder>statusOponent1</zorder>
     <zorder>statusOponent2</zorder>
     <zorder>nameOponent2</zorder>
     <zorder>countCoinsOponent2</zorder>
     <zorder>iconCoinsOponent2</zorder>
     <zorder>countCoinsOponent3</zorder>
     <zorder>iconCoinsOponent3</zorder>
     <zorder>nameOponent3</zorder>
     <zorder>oponent3Card1</zorder>
     <zorder>oponent3Card2</zorder>
     <zorder>statusOponent3</zorder>
     <zorder>playerCard2</zorder>
     <zorder>playerCard1</zorder>
     <zorder>playerCard1BTN</zorder>
     <zorder>playerCard2BTN</zorder>
     <zorder>card5BTN</zorder>
     <zorder>card1BTN</zorder>
     <zorder>card3BTN</zorder>
     <zorder>card2BTN</zorder>
     <zorder>card4BTN</zorder>
     <zorder>restructuringBTN</zorder>
     <zorder>takeOneMoneyBTN</zorder>
     <zorder>checkBTN</zorder>
     <zorder>notCheckBTN</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
